WrightTools/data.py

```diff
@@ -2861,9 +2861,79 @@ def from_shimadzu(filepath, name=None, verbose=True):
     return data
 
 
+def from_1130(filepath, name=None, delimiter=None, verbose=True):
+    """Create a data object from a SPC-130 TCSPC an exported
+    comma-delimited (.asc) file within the SPCM 9.75 software.
+
+    Parameters
+    ----------
+    filepath : string
+        Path to SPC-130 output file (.asc).
+    name : string (optional)
+        Name to give to the created data object. If None, filename is used.
+        Default is None.
```

Comment:
please document delimiter kwarg

Reply:
also if the default is `','` it's probably best to put that into the kwarg (instead of `None`)